site/content/get-started.md

```diff
@@ -86,8 +88,9 @@ make create-kind-cluster
 Use `kubectl` to add the API resources for NGINX Gateway Fabric with the following command:
 
 ```shell
-kubectl kustomize "https://github.com/nginxinc/nginx-gateway-fabric/config/crd/gateway-api/standard?ref=v1.4.0" | kubectl apply -f -
+kubectl kustomize "https://github.com/nginxinc/nginx-gateway-fabric/config/crd/gateway-api/standard?ref=v1.5.0" | kubectl apply -f -
 ```
+
```

Comment:
have made similar edits along the file due to lint errors to have blank space between code blocks.

Reply:
we weren't seeing these issues before in the last docs PR, why did they pop up now?

Reply:
the `<ghcode>` shortcode is nice because it displays the yaml to the user. To fix the bare URL error can you try putting the URL in angle brackets or back-ticks: https://github.com/updownpress/markdown-lint/blob/master/rules/034-no-bare-urls.md?

Reply:
This seems to have fixed it. Thank you soo much for the help @kate-osborn